gitlab-pipeline/stage/init.yml

```diff
@@ -1,9 +1,11 @@
+variables:
+  GIT_SUBMODULE_DEPTH: 1
```

Comment:
This only affect GitLab clones, so in this context it won't help with the `git clone` commands in this job. Right?

Reply:
Right - of course. I might have misread the errors as well, it seems to work now - so I think it's sufficient to just remove this and keep the hetzner runner and the dependency proxy. The git submodule command already has the depth set to 1